### DISCUSSION TITLE
Toast Align Message Top

### DISCUSSION THREAD

**Discussion by wortkotze** [wix/react-native-ui-lib#1788] [Q&A] [answered]
Hi,

sry for a dump question. I don't get the toast message aligned top in the message box :(
is there any way ?

thx
Stephan

**Comment by ethanshar** [comment, marked as answer]
Hi @wortkotze 

Are you using the `Incubator.Toast` component? Have you tried passing the `position` prop? 

Can you share more info on the issue.
